src/mocks/components/MockRichTextField.tsx

```diff
@@ -0,0 +1,25 @@
+import { camelCase } from "change-case";
+import { useState } from "react";
+import { RichTextFieldProps } from "src/components/RichTextField";
+import { useTestIds } from "src/utils";
+
+export function RichTextField(props: RichTextFieldProps) {
```

Comment:
so been working to try and find a cute way to abstract the Jest mocks into `Beam` itself but have not been successful.

The mocks are being exported alongside the components in Beam but this means you still need to manually define each mock in your application's `setupTests.js` such as:

```
jest.mock("@homebound/beam/dist/inputs/DateField", () => {
  return require("@homebound/beam/dist/inputs/DateField.mock");
});

jest.mock("@homebound/beam/dist/inputs/SelectField", () => {
  return require("@homebound/beam/dist/inputs/SelectField.mock");
});

jest.mock("@homebound/beam/dist/inputs/MultiSelectField", () => {
  return require("@homebound/beam/dist/inputs/MultiSelectField.mock");
});
```

As a next iteration, it would be great if we could do these mocks automatically. The idea is to export a function from Beam containing all jest mocks and then in application's `setupTests.js` we could just import and execute this function.